docs/notebooks/inequality_constraints.pct.py

```diff
@@ -11,7 +11,7 @@
 # %% [markdown]
 # ## The problem
 #
-# In this tutorial, we replicate one of the results of <cite data-cite="gardner14"/>, specifically their synthetic experiment "simulation 1", which consists of an objective function with a single constraint, defined over a two-dimensional input domain. We'll start by defining the problem parameters.
+# In this tutorial, we replicate one of the results of [[GKZ+14](https://secondmind-labs.github.io/trieste/autoapi/trieste/index.html#id4)], specifically their synthetic experiment "simulation 1", which consists of an objective function with a single constraint, defined over a two-dimensional input domain. We'll start by defining the problem parameters.  The constraint is satisfied when `constraint(input_data) <= threshold`.
```

Comment:
these hyperlinks will link to the bibliography on the remote site, while they should point to the bibliography in the current doc build

Reply:
Yes, open to alternatives, but it wasn't working as it was and was also breaking the formatting in the notebook.

Reply:
is it sufficient to hardlink to the paper:
http://proceedings.mlr.press/v32/gardner14.html
?

Reply:
that would lose the benefit the sphinxcontrib-bibtex plugin, which I'm not keen to lose so quickly. I'm looking at the syntax for the `<cite data-cite=...>`. Perhaps there's a way of doing that differently that will work nicely.

I've prioritised the docs website over the notebooks so far, having assumed that's what people will see first and will use more often

Reply:
this page shows how to do citations in jupyter https://nbviewer.jupyter.org/github/ipython/nbconvert-examples/blob/master/citations/Tutorial.ipynb

Reply:
`<cite data-cite="gardner14">Gardner et al.</cite>` keeps the link as is in the docs, but shows _`Gardner et al.`_ in the notebook, which is an improvement but not perfect

Reply:
you can do the whole thing with
```
<cite data-cite="gardner14">[Gardner et al.](http://proceedings.mlr.press/v32/gardner14.html)</cite>
```